packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -1137,67 +1146,135 @@ function CheckoutLineItem( {
 			{ isJetpackSearch( product ) && <JetpackSearchMeta product={ product } /> }
 
 			{ isEmail && <EmailMeta product={ product } isRenewal={ isRenewal } /> }
-
-			{ children }
-
-			{ hasDeleteButton && removeProductFromCart && (
+			{ ! hasCheckoutVersion( '2' ) && (
```

Comment:
I wonder if there's some way we can do this without having to duplicate everything? 🤔  What are the differences between the two versions of this block? I see the `Remove` button renamed but everything else looks the same? Oh, is it just the `BillingLine` wrapper? If so, I bet we can make those styles optional rather than duplicating this whole block.

Reply:
I think I tried doing this in an earlier version but the wrapper existing was causing some odd behavior with the original checkout.

Reply:
Actually, I found that the `display: inherit` is what broke this dropdown in regular checkout, changing to `display: block` fixes it.